nncf/quantization/layers.py

```diff
@@ -307,22 +311,40 @@ def run_export_quantization(self, x: torch.Tensor):
             input_high = input_range
 
             if self._export_mode == QuantizerExportMode.ONNX_QUANTIZE_DEQUANTIZE_PAIRS:
-                y_scale, y_zero_point = get_scale_zp_from_input_low_input_high(self.level_low,
-                                                                               self.level_high,
-                                                                               input_low,
-                                                                               input_high)
+                if self.is_saturation_fix:
+                    y_scale, y_zero_point = get_scale_zp_from_input_low_input_high(2 * self.level_low,
+                                                                                   2 * self.level_high + 1,
+                                                                                   2 * input_low,
+                                                                                   127. / 63. * input_high)
+                else:
+                    y_scale, y_zero_point = get_scale_zp_from_input_low_input_high(self.level_low,
+                                                                                   self.level_high,
+                                                                                   input_low,
+                                                                                   input_high)
 
         if self._export_mode == QuantizerExportMode.ONNX_QUANTIZE_DEQUANTIZE_PAIRS:
             if self.per_channel:
                 if torch.allclose(y_scale - y_scale[0], torch.zeros_like(y_scale)) and torch.allclose(
                         y_zero_point - y_zero_point[0], torch.zeros_like(y_zero_point)):
                     y_scale, y_zero_point = y_scale[0], y_zero_point[0]
+                    # TODO:: do we need Clip?
                     return ExportQuantizeToONNXQuantDequant.apply(x, y_scale, y_zero_point)
                 raise RuntimeError("PyTorch v1.5.0 export to ONNX using QuantizeLinear-DequantizeLinear "
                                    "doesn't support per channel quantization")
             return ExportQuantizeToONNXQuantDequant.apply(x, y_scale, y_zero_point)
+
         if self._export_mode == QuantizerExportMode.FAKE_QUANTIZE:
-            return ExportQuantizeToFakeQuantize.apply(x, self.levels, input_low, input_high, input_low, input_high)
+            # TODO:: do we need Clip?
+            scale_low = 1
+            scale_high = 1
+            if self.is_saturation_fix:
+                scale_low = 2
+                scale_high = 127. / 63.
```

Comment:
Same here

Reply:
Agree. Please take a look at the new version